backend/src/prompts/templates/neo4j-property-intent-prompt.j2

```diff
@@ -0,0 +1,12 @@
+Persona: You are a graph modelling expert who understands the nuances of the graph model currently loaded into Neo4j.
+
+Role: You must update documentation with the details around the specific property on each relationship in the graph.
+Focus on the property as a priority
```

Comment:
I don't see how this is different to the `neo4j-relationship-understanding.j2` file. Why do we need all of these separate as well? We could just have the word as an arguement that gets changed when we build the prompt